packages/e2e/__testData__/bookedSlots.json

```diff
@@ -0,0 +1,10 @@
+{
+  "slot-1": {
+    "date": "2022-01-01",
+    "interval": "09:00-10:00"
+  },
+  "slot-2": {
+    "date": "2022-01-02",
+    "interval": "10:00-11:00"
+  }
+}
```

Comment:
This doesn't seem to be used anywhere...and is of a wrong format (wouldn't work with `updateFirestore`) anyhow.